kuma/users/admin.py

```diff
@@ -23,20 +24,29 @@ class UserAdmin(BaseUserAdmin):
     Extends the admin view of users to show date_joined field
     add a filter on the field too
     """
-    list_display = ('username', 'fullname', 'email', 'title', 'organization',
-                    'location', 'content_flagging_email', 'tags',
+    list_display = ('username', 'fullname', 'email',
+                    'bio', 'website', 'revisions',
                     'date_joined', 'is_staff', 'is_active')
-    list_editable = ('content_flagging_email', 'tags')
     list_filter = ('is_staff', 'is_superuser', 'is_active', 'date_joined')
     ordering = ('-date_joined',)
     search_fields = ('username', 'homepage', 'title', 'fullname',
                      'organization', 'location', 'bio', 'email', 'tags__name')
 
-    formfield_overrides = {
-        NamespacedTaggableManager: {
-            'widget': TagWidget(attrs={'size': 45})
-        }
-    }
+    def revisions(self, obj):
+        """HTML link to user's revisions with count""" 
+        link = reverse('dashboards.revisions') +\
+            ('?user=%s' % obj.username)
+        count = obj.wiki_revisions().count()
+        return ('<a href="%(link)s"><strong>%(count)s</strong></a>' %
+            {'link': link, 'count': count})
 
+    revisions.allow_tags = True
+
+    def website(self, obj):
+        """HTML link to user's website""" 
+        return ('<a href="%(link)s"><strong>%(url)s</strong></a>' %
```

Comment:
If this is valuable, I need to verify whether the content of the field is sanitized. Otherwise, this presents an attack vector.